<?xml version="1.0" encoding="UTF-8"?>

<launch>
	<node pkg="ArmduinoRover" type="odometry_node" name="odometry">
	</node>

	<node pkg="ArmduinoRover" type="diff_tf.py" name="diff_odom">
	<param name="rate" value="2.0" />
	<param name="ticks_meter"  value="93" />
	<param name="base_width" value="0.137" />
	</node>

</launch>
